gaps/gap-draft_mid_agreement_payments/gap-draft_mid_agreement_payments.md

```diff
@@ -0,0 +1,91 @@
+---
+gap: <to be assigned>
+title: Mid Agreement payments
+description: Specification of Agreement standard for negotiating and describing continues payments during task execution.  
+author: nieznany.sprawiciel (@nieznanysprawiciel), stranger80 (@stranger80)>, shadeofblue (@shadeofblue)
+status: Draft
+type: Standards
+---
+
+## Abstract
+Introduces Agreement properties standard to allow negotiating incremental payments during task execution.
+
+## Motivation
+Current payments scheme allows only to pay for tasks, after all computations are finished and Invoice is sent to Requestor.
+In case of long-running tasks or services, Provider would expect, that he will be paid during
+execution, because he doesn't want to wait for task finish.
+
+## Specification
+
+Setting `due_date` in Debit Note will cause yagna payment module to automatically pay for Debit Note,
+after it was accepted. This specification describes, how Requestor and Provider should negotiate payments parameters.
+
+### New properties
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `golem.com.scheme.payu.debit-note-interval-sec?` | Number (uint32) [Negotiable] | Minimum time (in seconds) between subsequent Debit Notes. The Provider must not issue the next Debit Note before the interval period. The Requestor ignores Debit Notes issued before interval elapses. |
+| `golem.com.scheme.payu.payment-timeout-sec?` | Number (uint32) [Negotiable] | Maximum time to receive payment for any Debit Note. This time is counted from Debit Note’s issue date. |
+
+- Requestor can ignore Debit Note, if it came before expected interval
```

Comment:
But still payments module can get mad. Debit Note will be accepted at the same time, next Debit Note will. So payment driver have to deal with previous Debit Note due date either